index.html

```diff
@@ -0,0 +1,183 @@
+          and basic control-flow constructs which was the reason why it was
+          renamed from LiveScript to JavaScript.
+        </p>
+        <p>
+          In contrast to Java's compile-time system of classes built by
+          declarations, JavaScript supports a runtime system based on a small
+          number of data types representing numeric, Boolean, and string
+          values. JavaScript has a prototype-based object model instead of the
+          more common class-based object model. The prototype-based model
+          provides dynamic inheritance; that is, what is inherited can vary
+          for individual objects. JavaScript also supports functions without
+          any special declarative requirements. Functions can be properties of
+          objects, executing as loosely typed methods.
+        </p>
+
+        <p>
+          JavaScript is a very free-form language compared to Java. You do not
+          have to declare all variables, classes, and methods. You do not have
+          to be concerned with whether methods are public, private, or
+          protected, and you do not have to implement interfaces. Variables,
+          parameters, and function return types are not explicitly typed.
+        </p>
+      </section>
+      <section class="main-section" id="Hello_world">
+        <header>
+          <h1>Hello world</h1>
+        </header>
+
+        <p>
+          To get started with writing JavaScript, open the Scratchpad and
+          write your first "Hello world" JavaScript code:
+        </p>
+
+        <p>
+          Select the code in the pad and hit Ctrl+R to watch it unfold in your
+          browser!
+        </p>
+        <div class="content">
+          <p>function greetMe(yourName) { alert("Hello " + yourName); }</p>
+          <p>greetMe("World");</p>
+        </div>
+      </section>
+      <section class="main-section" id="Variables">
+        <header>
+          <h1>Variables</h1>
+        </header>
+        <p>
+          You use variables as symbolic names for values in your application.
+          The names of variables, called identifiers, conform to certain
+          rules.
+        </p>
+        <p>
+          A JavaScript identifier must start with a letter, underscore (_), or
+          dollar sign ($); subsequent characters can also be digits (0-9).
+          Because JavaScript is case sensitive, letters include the characters
+          "A" through "Z" (uppercase) and the characters "a" through "z"
+          (lowercase).
+        </p>
+
+        <p>
+          You can use ISO 8859-1 or Unicode letters such as å and ü in
+          identifiers. You can also use the Unicode escape sequences as
+          characters in identifiers. Some examples of legal names are
+          Number_hits, temp99, and _name.
+        </p>
+      </section>
+    </div>
+    <nav id="navbar">
+      <header>
+        <h1>JS Documentation</h1>
```

Comment:
Please remember to observe the correct HTML semantics in tags. The h1 tag being the most important and h6 being the least important.

Reply:
It is important to always state the why a particular change is being requested in order to help the student learn from a comment.